### PR TITLE
Fix audio filepath error

### LINKED ISSUE #237: Song does not play when game is executed outside repository folder
## Describe the bug
Song does not play when game is executed outside repository folder. Only when you call `ttsnake` in snaskii21 folder it plays.

## Steps to Reproduce
1. Execute game in repository folder
2. Then execute it in any other folder

## What the expected behavior should've been
The song should be reproduced whatever the directory you execute the binary.

#

### PR DESCRIPTION
**Pull request by l-a-motta**
Fixes #237 .

During local testing it worked, but it might need further testing. The change to use a URL with an HTTP request is still more interesting than this solution, and I suggest that one be pursued in favour of this quick fix.

### CODE REVIEW

File: src/ttsnake.c [hunk 1]
```diff
@@ -323,8 +323,8 @@ void showscene (scene_t* scene, int scene_type, int menu) {
 
 /* Initialize resources and counters. */
 void init_game () {
-	
-    system("mpg123 --no-visual --no-control --quiet ./sound/maintheme.mp3 &");
+	/* Possible solution to filepath: either use path as 'sound/maintheme.mp3' or use HTTP protocol from mpg123 (https://www.mankier.com/1/mpg123)*/
```
Comment (Idalen) [inline src/ttsnake.c]: This comment is no longer needed.

File: src/ttsnake.c [hunk 2]
```diff
@@ -887,6 +887,7 @@ int main (int argc, char **argv) {
     free(curr_data_dir);
     free(snake.positions);
 
+    /* Be careful with the killall or we might leave the program without enough time to end, resulting in infinite music */
```
Comment (Idalen) [inline src/ttsnake.c]: You should open a issue to use a proper instruction instead the current one.

**Review by Idalen — CHANGES_REQUESTED** [review]

**Comment by l-a-motta** [comment]
Music is now received by HTTP request. Comments removed

File: src/ttsnake.c (hunk 1 above)
Comment (l-a-motta) [reply src/ttsnake.c]: Agreed

File: src/ttsnake.c (hunk 2 above)
Comment (l-a-motta) [reply src/ttsnake.c]: It might not be necessary anymore with the HTTP request being done. It will certainly die now, I think

**Review by l-a-motta — COMMENTED** [review]
New change looks good

**Review by Idalen — APPROVED** [review]
Nice job!

**Pull request merged**